bitnami/redis/Chart.yaml

```diff
@@ -25,4 +25,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 17.12.0
+version: 17.12.1
```

Comment:
As this is a new feature we would bump a minor version following [semver](https://semver.org/): `17.13.0`. Could you update it?

Reply:
@aoterolorenzo done